README.md

```diff
@@ -1,19 +1,88 @@
-# Exodus - Android Application
-
-[![Build Status](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml/badge.svg?branch=master)](https://github.com/Exodus-Privacy/exodus-android-app/actions/workflows/main.yml) [![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/Exodus-Privacy/exodus-android-app.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Exodus-Privacy/exodus-android-app/context:java)
+<h1 align="center">Exodus - Android Application</h1>
```

Comment:
**suggestion**: Not a big fan of using html here, if we can keep using markdown syntax I think that would be easier to maintain

Reply:
```suggestion
<p align="center">
# Exodus - Android Application
</p>
```

Is it better like this, i don't have find other way to centered text in markdown

Reply:
Yes, HTML is the only option I think

Reply:
I don't really like using the HTML format here, but I have to admit it looks good, so why not ;)